fastddsspy_tool/test/application/test_class.py

```diff
@@ -208,6 +209,7 @@ def valid_output(self, output) -> bool:
         lines_expected_output = expected_output.splitlines()
         lines_output = output.splitlines()
 
+        # TODO (Raul): If guid and rate are on the same line this will not work.
```

Comment:
Address?

Reply:
This should be addressed in another PR, as I detected it here, but it has remained this way since the test implementation.